L1Trigger/Phase2L1ParticleFlow/python/l1pfJetMet_cff.py

```diff
@@ -0,0 +1,38 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoMET.METProducers.PFMET_cfi import pfMet as _pfMet
+_pfMet.calculateSignificance = False
+l1PFMetCalo    = _pfMet.clone(src = "l1pfCandidates:Calo")
+l1PFMetPF      = _pfMet.clone(src = "l1pfCandidates:PF")
+l1PFMetPuppi   = _pfMet.clone(src = "l1pfCandidates:Puppi")
+
+l1PFMets = cms.Sequence(l1PFMetCalo + l1PFMetPF + l1PFMetPuppi)
+
+from RecoJets.JetProducers.ak4PFJets_cfi import ak4PFJets as _ak4PFJets
+_ak4PFJets.doAreaFastjet = False
+ak4PFL1Calo    = _ak4PFJets.clone(src = 'l1pfCandidates:Calo')
+ak4PFL1PF      = _ak4PFJets.clone(src = 'l1pfCandidates:PF')
+ak4PFL1Puppi   = _ak4PFJets.clone(src = 'l1pfCandidates:Puppi')
+
+_correctedJets = cms.EDProducer("L1TCorrectedPFJetProducer", 
+    jets = cms.InputTag("_tag_"),
+    correctorFile = cms.string("L1Trigger/Phase2L1ParticleFlow/data/jecs/jecs.PU200.root"),
+    correctorDir = cms.string("_dir_"),
+    copyDaughters = cms.bool(False)
+)
+# Using phase2_hgcalV10 to customize the config for all 106X samples, since there's no other modifier for it
```

Comment:
this is because things like JECs really ought to be specified in GTs

Reply:
Will take that as a suggestion for the future, but I can't do it for this PR (there's plenty of other L1T "conditions" that are implemented in configuration or text files)